scripts/spack/packages/axom/package.py

```diff
@@ -120,6 +121,10 @@ class Axom(CachedCMakePackage, CudaPackage):
         depends_on('umpire cuda_arch={0}'.format(sm_),
                    when='+umpire cuda_arch={0}'.format(sm_))
 
+    # SERAC EDIT BEGIN - we don't have or use the c2c package
+    #depends_on("c2c", when="+c2c")
+    # SERAC EDIT BEGIN
```

Comment:
Will we be able to get rid of this when we update our Spack version?

Reply:
this errors because there wasnt a c2c package in spack.  This hasnt been pushed up because it only lives on the rz atm.